packages/react-native-otlp/package.json

```diff
@@ -4,7 +4,13 @@
   "description": "A React Native wrapper for the Embrace SDK that allows to connect to OTLP endpoints",
   "files": [
     "lib",
-    "android",
+    "android/gradle",
```

Comment:
are we sure we need these?
* android/gradle
* android/gradlew
* android/gradlew.bat

the current core package doesn't include them, I think it'd be the customer's app's gradle that would get used when building